CMakeLists.txt

```
@@ -192,11 +192,16 @@ endif(MSVC)
 
 set(PLASMA_SOURCE_ROOT "${CMAKE_CURRENT_SOURCE_DIR}/Sources/Plasma")
 set(PLASMA_TOOLS_ROOT "${CMAKE_CURRENT_SOURCE_DIR}/Sources/Tools")
+set(PLASMA_MAX_PLUGIN_ROOT "${CMAKE_CURRENT_SOURCE_DIR}/Sources/MaxPlugin")
 
 add_subdirectory(Scripts)
 add_subdirectory(Sources/Plasma)
 add_subdirectory(Sources/Tools)
```

The entire Tools directory can be made conditional now, like the MaxPlugin one:

```suggestion

if(PLASMA_BUILD_TOOLS)
    add_subdirectory(Sources/Tools)
endif()
```

(and then remove the `if(PLASMA_BUILD_TOOLS)` condition in Sources/Tools/CMakeLists.txt)